src/main/resources/public/js/date.time.utils.js

```diff
@@ -28,7 +28,8 @@
     }
 
     function getTimePart(part) {
-      return part < 1 ? '00' : (part < 10 ? `0${part}` : part);
+      const timePart = part < 1 ? '00' : part;
+      return part < 10 ? `0${part}` : timePart;
```

Comment:
IMO this change makes the expression overcomplicated -- the original form is better readable. If `part < 1` than we don't need to evaluate `part < 10` at all...